src/Fixer/Strict/StrictTypesFixer.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Strict;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ */
+final class StrictTypesFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        if (!$tokens->isMonolithicPhp()) {
+            return;
+        }
+
+        $declareTokens = array(
+            new Token(array(T_OPEN_TAG, '<?php ')),
+            new Token(array(T_DECLARE, 'declare')),
+            new Token('('),
+            new Token(array(T_STRING, 'strict_types')),
+            new Token('='),
+            new Token(array(T_LNUMBER, 1)),
```

Comment:
The `1` is the problem of the failing test. The content should be a string, i.e. change it to `'1'` and you should be OK :)

I'll add a note to my TODO list to make the output of the failing test better (i.e. it would be nice if the output of the `toJson` method of `Token` would show `'1'` as content and not `1`)